Manito/Manito/Screens/Detail-Wait/DetailEditViewController.swift

```diff
@@ -18,16 +19,22 @@ final class DetailEditViewController: UIViewController {
     // MARK: - property
     
     private let editMode: DetailEditView.EditMode
-    private let detailRoomRepository: DetailRoomRepository = DetailRoomRepositoryImpl()
-    private let room: RoomInfo
+    private var roomPublisher: CurrentValueSubject<CreatedRoomInfoRequestDTO, Never>
+    private let viewModel: any BaseViewModelType
     weak var detailWaitDelegate: DetailWaitViewControllerDelegate?
+    private var cancellable = Set<AnyCancellable>()
+    private var viewModelOutput: DetailEditViewModel.Output?
     
     // MARK: - init
     
-    init(editMode: DetailEditView.EditMode, room: RoomInfo) {
+    init(editMode: DetailEditView.EditMode, room: RoomInfo, viewModel: any BaseViewModelType) {
```

Comment:
p5;
가독성과 저희 코딩 컨벤션에 맞게 개행리 부탁드립니다!
```suggestion
    init(editMode: DetailEditView.EditMode,
         room: RoomInfo,
         viewModel: any BaseViewModelType)
```

Reply:
넵! 예쁘게 수정했습니다!
[59dcf52](https://github.com/DeveloperAcademy-POSTECH/MC2-Team5-Firefighter/pull/600/commits/59dcf52f028b0ed9d38a2718c62971ebc2475355)